AndorsTrail/res/raw/conversationlist_stoutford.json

```diff
@@ -885,7 +885,8 @@
                     {
                         "requireType":"questProgress",
                         "requireID":"rumblings",
-                        "value":50
+                        "value":50,
+                        "negate":true
```

Comment:
I'll look at them. I don't recall changing these, and can't think why I would have.